archivers/par2/Portfile

```diff
@@ -17,20 +17,18 @@ license           GPL-2
 maintainers       jabber.org:julian
 categories        archivers
 platforms         darwin 
-homepage          http://parchive.sourceforge.net/
-master_sites      sourceforge:parchive
+homepage          https://github.com/Parchive/par2cmdline
+master_sites      ${homepage}/releases/download/v${version}
```

Comment:
this should use the "github" PortGroup; several fields are set then correctly automatically.

Reply:
Done